lib/screens/common_widgets/contact_initial.dart

```diff
@@ -39,7 +39,7 @@ class ContactInitial extends StatelessWidget {
       ),
       child: Center(
         child: Text(
-          initials!.substring(0, index).toUpperCase(),
+          initials!.substring(1, index).toUpperCase(),
```

Comment:
Why are we initial index from 0 to 1 ?
The contact initial for atSign(@46honest) is "6h" instead of "46" now